cmd/simulator/config/flags.go

```diff
@@ -36,26 +38,30 @@ var (
 )
 
 type Config struct {
-	Endpoints    []string      `json:"endpoints"`
-	MaxFeeCap    int64         `json:"max-fee-cap"`
-	MaxTipCap    int64         `json:"max-tip-cap"`
-	Workers      int           `json:"workers"`
-	TxsPerWorker uint64        `json:"txs-per-worker"`
-	KeyDir       string        `json:"key-dir"`
-	Timeout      time.Duration `json:"timeout"`
-	BatchSize    uint64        `json:"batch-size"`
+	Endpoints       []string      `json:"endpoints"`
+	MaxFeeCap       int64         `json:"max-fee-cap"`
+	MaxTipCap       int64         `json:"max-tip-cap"`
+	Workers         int           `json:"workers"`
+	TxsPerWorker    uint64        `json:"txs-per-worker"`
+	KeyDir          string        `json:"key-dir"`
+	Timeout         time.Duration `json:"timeout"`
+	BatchSize       uint64        `json:"batch-size"`
+	BlockchainIDStr string        `json:"blockchainIDStr"`
+	Metrics         string        `json:"metrics"`
```

Comment:
can we change this to `[]string` so that we can gather metrics from multiple nodes when relevant same as hitting multiple rpc endpoints?

Could we also rename to `metrics-endpoints`

Reply:
sure